spec/controllers/votes_controller_spec.rb

```diff
@@ -5,17 +5,27 @@
   include TestFactories
   include Devise::TestHelpers
 
+  before do
+    request.env["HTTP_REFERER"] = '/'
+    @user = authenticated_user
+    @post = associated_post
+    sign_in @user
+  end
+
   describe '#up_vote' do
     it "adds an up-vote to the post" do
-      request.env["HTTP_REFERER"] = '/'
-      @user = authenticated_user
-      @post = associated_post
-      sign_in @user
-
       expect {
         post( :up_vote, post_id: @post.id )
       }.to change{ @post.up_votes }.by 1
     end
   end
 
+  describe '#down_vote' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.